frontend/playground-frontend/package.json

```diff
@@ -10,13 +10,20 @@
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.9.3",
     "@reduxjs/toolkit": "^1.8.5",
+    "@testing-library/react": "^13.4.0",
```

Comment:
nice we are using this library. at my work, we are migrating our JS unit tests to RTL from enzyme